### PR TITLE
style: Added encoding in headers

### PR DESCRIPTION
**Pull request by frgfm**
This PR aims at:
- adding utf8 encoding when it was missing in header to avoid issues
- switched to Python 3 class definitions style
- switched from absolute to relative import when it was not needed (absolute could use existing package installation even if they do not share the same codebase)

### CODE REVIEW

File: pyronear/datasets/wildfire/fire_labeler.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+# -*- coding: utf-8 -*-
+
 import re
 from itertools import combinations
 
 import numpy as np
 
 
-class FireLabeler:
+class FireLabeler(object):
```
Comment (x0s) [inline pyronear/datasets/wildfire/fire_labeler.py]: All classes are new-style in python 3, so we no longer need to inherit from object.

File: pyronear/datasets/wildfire/wildfire.py [hunk 2]
```diff
@@ -72,7 +75,7 @@ def _get_targets(self, index):
         return torch.from_numpy(self.metadata[self.target_names].iloc[index].values)
 
 
-class WildFireSplitter:
+class WildFireSplitter(object):
```
Comment (x0s) [inline pyronear/datasets/wildfire/wildfire.py]: Same all classes are new-style in python 3, so we no longer need to inherit from object.

File: pyronear/datasets/wildfire/fire_labeler.py [hunk 3]
```diff
@@ -1,10 +1,13 @@
+#!usr/bin/python
```
Comment (x0s) [inline pyronear/datasets/wildfire/fire_labeler.py]: Why do you add the shebang ? this file is not intended to be used as a script

Comment (x0s) [reply pyronear/datasets/wildfire/fire_labeler.py]: same remark for all new shebang in the current PR

**Review by x0s — CHANGES_REQUESTED** [review]
Thanks for the PR. Quickly:
- Class definition are already new-style
- The files concerned in this PR are not intended to be used as scripts
- Okay for relative imports

File: pyronear/datasets/wildfire/fire_labeler.py (hunk 1 above)
Comment (frgfm) [reply pyronear/datasets/wildfire/fire_labeler.py]: My bad, you are very correct

File: pyronear/datasets/wildfire/fire_labeler.py (hunk 3 above)
Comment (frgfm) [reply pyronear/datasets/wildfire/fire_labeler.py]: Oops, always copy pasted it with the encoding, you are correct!

**Comment by frgfm** [comment]
> Thanks for the PR. Quickly:
> 
> * Class definition are already new-style
> * The files concerned in this PR are not intended to be used as scripts
> * Okay for relative imports

Thanks for the review, just made the suggested modifications!

**Review by x0s — APPROVED** [review]

**Pull request merged**